src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    static final int ITERATIONS = 3;
```

Comment:
If the constant is used only in this class, it must be private
```suggestion
    private static final int ITERATIONS = 3;
```